packages/redux-dynamic-modules/src/DynamicModuleLoader.tsx

```diff
@@ -38,14 +43,61 @@ export class DynamicModuleLoader<
     context: IDynamicModuleLoaderContext
   ) {
     super(props, context);
-    const { modules } = props;
-    const store: IModuleStore<any> = context.store;
-    if (!store) {
-      throw new Error("Could not load store from React context.");
+    if (!ReactReduxContext) {
+      if (!context.store) {
+        throw new Error("Could not load store from React context.");
+      }
+    }
+  }
+
+  /**
+   * Render a Redux provider
+   */
+  public render(): React.ReactNode {
+    if (ReactReduxContext) {
+      return (
+        <ReactReduxContext.Consumer>
+          {(context) => (
+            <DynamicModuleLoaderImpl
+              store={context.store}

```

Comment:
Do we need to assert here that ```context.store``` is not null?

Reply:
Yeah, thanks